database/migrations/2025_01_29_224410_create_list_movie_table.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateListMovieTable extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('list_movie', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('list_id')->constrained()->onDelete('cascade');
```

Comment:
```suggestion
            $table->foreignId(MovieList::class, 'list_id')->constrained()->onDelete('cascade');
```

samma här, tror vi behöver specifiera 'list_id' annars blir det 'movie_list_id'